src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -296,10 +296,12 @@ mod tests {
     use assert_matches::assert_matches;
     use risingwave_hummock_sdk::compaction_group::StaticCompactionGroupId;
     use risingwave_hummock_sdk::CompactionGroupId;
+    use risingwave_pb::hummock::compact_task::TaskStatus;
```

Comment:
Looks like this import is unused with `failpoints` disabled. We may put it in `async fn test_failpoints()`.